webapp/lib/view.dart

```diff
@@ -113,6 +115,130 @@ class DashboardView {
   }
 }
 
+enum FormGroupTypes {
+  TEXT,
+  CHECKBOX,
+  DATE
+}
+
+class ProjectConfigurationView {
+  DivElement configurationViewElement;
+  FormElement _projectConfigurationForm;
+
+  ProjectConfigurationView() {
+    configurationViewElement = new DivElement()
+      ..classes.add('project-configuration');
+    _projectConfigurationForm = new FormElement()
+      ..classes.add('configuration-form');
+    _buildForm();
+    configurationViewElement.append(_projectConfigurationForm);
+  }
+
+  void _buildForm() {
+    _projectConfigurationForm
+      ..append(
+        _multipleFormGroup('Project Languages:',
+          {'English': {'send': FormGroupTypes.CHECKBOX, 'receive': FormGroupTypes.CHECKBOX },
+            'Somali': {'send': FormGroupTypes.CHECKBOX, 'receive': FormGroupTypes.CHECKBOX }})
+      )
+      ..append(
+        _singleFormGroup('Automated translations enabled', FormGroupTypes.CHECKBOX)
+      )
+      ..append(
+        _singleFormGroup('Coda dataset regex', FormGroupTypes.TEXT)
+      )
+      ..append(
+        _singleFormGroup('Rapidpro token', FormGroupTypes.TEXT)
+      )
+      ..append(
+        _singleFormGroup('Project start date', FormGroupTypes.DATE)
+      )
+      ..append(
+        _multipleFormGroup('User Configuration:',
+          {'person1@africanvoices.org:':
+            {'can see messages': FormGroupTypes.CHECKBOX,
+              'can perform translations': FormGroupTypes.CHECKBOX,
+              'can send messages': FormGroupTypes.CHECKBOX,
+              'can send custom messages': FormGroupTypes.CHECKBOX,
+              'can approve actions': FormGroupTypes.CHECKBOX,
+              'can configure the project': FormGroupTypes.CHECKBOX
+            },
+          'person2@africanvoices.org:':
+            {'can see messages': FormGroupTypes.CHECKBOX,
+              'can perform translations': FormGroupTypes.CHECKBOX,
+              'can send messages': FormGroupTypes.CHECKBOX,
+              'can send custom messages': FormGroupTypes.CHECKBOX,
+              'can approve actions': FormGroupTypes.CHECKBOX,
+              'can configure the project': FormGroupTypes.CHECKBOX
+            },
+          'person3@africanvoices.org:':
```

Comment:
nit: should be `...@africasvoices.org` - can you replace in all the other email addresses too?